webapi/src/main/scala/org/knora/webapi/OntologyConstants.scala

```diff
@@ -117,23 +117,17 @@ object OntologyConstants {
         Standoff.StandoffOntologyLabel
     )
 
-    /**
-      * IRIs of ontologies that we serve from Scala constants, rather than from the triplestore.
-      */
-    val ConstantOntologies = Set(
-        KnoraApiV2Simple.KnoraApiOntologyIri,
-        KnoraApiV2WithValueObjects.KnoraApiOntologyIri
-    )
-
     object KnoraBase {
         val KnoraBaseOntologyLabel: String = "knora-base"
         val KnoraBaseOntologyIri: IRI = KnoraInternal.InternalOntologyStart + KnoraBaseOntologyLabel
 
         val KnoraBasePrefix: String = KnoraBaseOntologyLabel + ":"
         val KnoraBasePrefixExpansion: IRI = KnoraBaseOntologyIri + "#"
 
+        val CanBeInstantiated: IRI = KnoraBasePrefixExpansion + "canBeInstantiated"
+        val IsEditable: IRI = KnoraBasePrefixExpansion + "isEditable"
+
         val Resource: IRI = KnoraBasePrefixExpansion + "Resource"
-        val ExternalResource: IRI = KnoraBasePrefixExpansion + "ExternalResource"
         val Representation: IRI = KnoraBasePrefixExpansion + "Representation"
         val AudioRepresentation: IRI = KnoraBasePrefixExpansion + "AudioRepresentation"
         val DDDRepresentation: IRI = KnoraBasePrefixExpansion + "DDDrepresentation"
```

Comment:
This should be `DDDRepresentation`

Reply:
It was also wrong in knora-base. This should fix it https://github.com/dhlab-basel/Knora/pull/775/commits/d1446090b09d96988031aca6601f840b4d1d8f04